documentation/LLC-quickstart.md

```diff
@@ -0,0 +1,401 @@
+    
+        // enable / disable watching file and executing tests whenever any file changes
+        autoWatch: false,
+    
+        // --no-sandbox allows our tests to run in Linux without having to change the system.
+        // --disable-web-security allows us to authenticate from the browser without having to write tests using interactive auth, which would be far more complex.
+        browsers: ["ChromeHeadlessNoSandbox"],
+        customLaunchers: {
+          ChromeHeadlessNoSandbox: {
+            base: "ChromeHeadless",
+            flags: ["--no-sandbox", "--disable-web-security"]
+          }
+        },
+    
+        // Continuous Integration mode
+        // if true, Karma captures browsers, runs the tests and exits
+        singleRun: false,
+    
+        // Concurrency level
+        // how many browser should be started simultaneous
+        concurrency: 1,
+    
+        browserNoActivityTimeout: 60000000,
+        browserDisconnectTimeout: 10000,
+        browserDisconnectTolerance: 3,
+        browserConsoleLogOptions: {
+          terminal: !isRecordMode()
+        },
+    
+        client: {
+          mocha: {
+            // change Karma's debug.html to the mocha web reporter
+            reporter: "html",
+            timeout: "600000"
+          }
+        }
+      });
+    };
+    ```
+1. **add test utils.**  
+
+    create a test/public folder and then copy the content [here](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/purview/purview-account-rest/test/public/utils) into public folder
+    
+    Now, you can add some sample tests like 
+    ```typescript
+    /*
+     * Copyright (c) Microsoft Corporation.
+     * Licensed under the MIT License.
+     *
+     * Code generated by Microsoft (R) AutoRest Code Generator.
+     * Changes may cause incorrect behavior and will be lost if the code is regenerated.
+     */
+    
+    import { Recorder } from "@azure-tools/test-recorder";
+    import * as assert from "assert";
+    import { createRecorder } from "./utils/recordedClient";
+    
+    describe("My test", () => {
+      let recorder: Recorder;
+    
+      beforeEach(async function() {
+        recorder = createRecorder(this);
+      });
+    
+      afterEach(async function() {
+        await recorder.stop();
+      });
+    
+      it("sample test", async function() {
+        console.log("Hi, I'm a test!");
+      });
+    });
+    ```
+    You may change the sample test into real tests to test against your libraries. 
+# How to write samples
+There're samples for TypeScript and JavaScript and for dev, You may copy the [samples folder](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/purview/purview-account-rest/samples) and [samples-dev folder](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/purview/purview-account-rest/samples-dev) and then change everything into your own services, including package-name, sample code, readme description etc.
+
+# How to create package
```

Comment:
We also need to mention that a `.env` file needs to be setup with the credentials to use during test. And that we need to set `TEST_MODE=record` to get the HTTP responses mocked and then set `TEST_MODE=playback` to run agains the mocked responses

Reply:
added. But I think the default TEST_MODE is playback ?

Reply:
I add the rushx pack step, because some service team may want a private artifact when they are still in private preview stage, with this hint, they don't need to ask us to send them an artifact.